tornettools/tornettools

```diff
@@ -288,13 +288,26 @@ def main():
         default=0.1)
 
     generate_parser.add_argument('-t', '--torperf_scale',
-        help="""A fractional number representing the ratio of the number of Perf clients to the
+        help="""DEPRECATED. Use torperf_n instead.
+            A fractional number representing the ratio of the number of Perf clients to the
             number of Tor clients that we emulate. E.g., a value of 0.001 means that we will run
             1 Perf client for every 1/0.001=1000 Tor clients. Perf clients help us benchmark Tor
             performance during an experiment and validate against Tor metrics.""",
         metavar="N", type=__type_fractional_float,
         action="store", dest="torperf_scale",
-        default=0.001)
+        default=None)
+
+    generate_parser.add_argument('--torperf_n',
```

Comment:
To make this user-facing option a bit more accessible, how about using `--torperf_num` instead (or  `--torperf_num_exit` and `--torperf_num_onionservice` if we end up creating separate knobs)?